infrastructure/docker-registry/README.md

```diff
@@ -1,74 +1,75 @@
+  ```yaml 
+  sentinel.enabled=true
+  ```
+To deploy Redis service, please see the link on the pre-requisites section.
+  
+### Postgres Configuration
+For the installation of Postgres service, please see the link on the pre-requisites section.
 
-  3. Configuration of the persistent disk. Please dimension this disk appropriately, as VM images may be rather large and hence you may fill up this space in a very short time.
-```yaml
-persistence:
-  accessMode: 'ReadWriteOnce'
-  enabled: true
-  size: 500Gi
-  storageClass: 'rook-ceph-block'
-```
+### Harbor Configuration
+The following outlines the most relevant modifications applied to the Harbor `values.yaml` file:
+
+  1. Annotations concerning the configuration of the Ingress controller. You have to set the type of the ingress (in our file values.yaml the type of the ingress is ingress) in the expose.type parameter.
+
+  2. The external URL to expose the Harbor Registry. To set the external URL you have to set the externalURL parameter
+
+  3. For the external redis service you have to modify the redis.type parameter to external and have to set the same master name of the previous redis service in redis.external.sentinelMasterSet parameter.
 
 ## Installing the Chart
-To install the chart, use the following command:
+Before installing the chart, the Harbor repository must be added to helm with the following command:
 ```bash
-$ helm install stable/docker-registry -f docker-registry-configuration.yaml -n docker-registry --generate-name
+helm repo add harbor https://helm.goharbor.io
+helm repo update
 ```
-Look [here](https://github.com/helm/charts/tree/master/stable/docker-registry#configuration) for a complete configuration guide.
-
-
-## Warning: max HTTP body size in ingress controller
-By default, the nginx ingress controller has a limit on the maximum size of the HTTP body. If the image to be uploaded is larger, an error is received (403). To avoid this problem, modify the configuration of your ingress controller and add a new annotation:
-```yaml
-nginx.ingress.kubernetes.io/proxy-body-size: "0"
+To install the chart with the release name `harbor` and apply the configuration specified by the `harbor-values.yaml` file, it is possible to proceed as follows:
+```bash
+$ helm install my-release harbor/harbor
 ```
+The above command install a release called 'my-release'. If you want install a release starting from a values.yaml file, you can use the following command:
+```bash
+$ helm install my-release harbor/harbor -f values.yaml
```

Comment:
Please, replace this generic commands with the complete one used to install/upgrade harbor.